.github/workflows/ci.yml

```diff
@@ -216,6 +216,7 @@ jobs:
             !:trino-hive,!:trino-orc,!:trino-parquet,
             !:trino-mongodb,!:trino-kafka,!:trino-elasticsearch,
             !:trino-redis,
+            !:trino-pulsar,
```

Comment:
Let's run pulsar together with redis or with kafka above. Your build takes just 4 minutes so I guess compilation takes the most.

Reply:
merged with Redis test, it'll take longer as BaseConnectorTest is added, but we can split later if needed.